finetuner/__init__.py

```diff
@@ -308,6 +308,69 @@ def get_token() -> str:
     return ft.get_token()
 
 
+def build_model(
+    name: str,
+    model_options: Dict[str, Any] = {},
+    batch_size: int = 32,
+    select_model: Optional[str] = None,
+    device: Optional[str] = None,
+    logging_level: str = 'DEBUG',
```

Comment:
lets remove this argument, it seems irrelevant. Also remove it from docstring

Reply:
which argument, logging_level?